sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -0,0 +1,203 @@
+---
+title: SonarScanner for MSBuild
+url: /analysis/scan/sonarscanner-for-msbuild/
+---
+
+[[info]]
+| **Download SonarQube Scanner for MSBuild 4.6.0.1930** - Compatible with SonarQube 6.7+ (LTS)  
+| By [SonarSource](https://www.sonarsource.com/) – GNU LGPL 3 – [Issue Tracker](https://github.com/SonarSource/sonar-scanner-msbuild/issues) – [Source](https://github.com/SonarSource-VisualStudio/sonar-msbuild-runner)   
+|
+| [.NET Framework 4.6+](https://github.com/SonarSource/sonar-scanner-msbuild/releases/download/4.6.0.1930/sonar-scanner-msbuild-4.6.0.1930-net46.zip) |
+| [.NET Core 2.0+](https://github.com/SonarSource/sonar-scanner-msbuild/releases/download/4.6.0.1930/sonar-scanner-msbuild-4.6.0.1930-netcoreapp2.0.zip) |
+| [.NET Core Global Tool](https://www.nuget.org/packages/dotnet-sonarscanner) 
+
+
+The SonarScanner for MSBuild is the recommended way to launch an analysis for projects/solutions using MSBuild or dotnet command as a build tool. It is the result of a [collaboration between SonarSource and Microsoft](http://www.sonarqube.org/announcing-sonarqube-integration-with-msbuild-and-team-build/). 
+
+SonarScanner for MSBuild is distributed as a standalone command line executable, as a extension for <!-- sonarcloud -->[Azure DevOps](/analysis/scan/sonarscanner-for-azure-devops/)<!-- /sonarcloud --><!-- sonarqube -->[Azure DevOps Server](/analysis/scan/sonarscanner-for-azure-devops/)<!-- /sonarqube -->, and as a plugin for [Jenkins](/analysis/scan/sonarscanner-for-jenkins/).
```

Comment:
The text representation of `Azure DevOpsAzure DevOps Server` isn't quite right - the words are just run together.  Perhaps add a forward slash between them?

Reply:
@duncanp-sonar they don't appear together in the same output. One version is used in SonarCloud docs and the other in SQ/static docs.

Reply:
Ok. In that case the content is incorrect. The SonarQube extension can be used in both Azure DevOps and Azure DevOps Server.

Reply:
Let's talk about this. Specifically, "can be used" versus "supported".